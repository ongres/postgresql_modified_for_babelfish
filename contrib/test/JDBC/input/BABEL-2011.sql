USE master;
GO

-- test volatility of functions in multi-row insert
-- newsequentialid()
CREATE TABLE myTable (ColumnA uniqueidentifier DEFAULT NEWSEQUENTIALID(), a int);
go
insert myTable (a) values (1), (2)
go
-- should be equal to 2
select count(distinct a) from myTable
go

-- newid()
CREATE TABLE t1_2011 (a int);
CREATE TABLE myTable2 (ColumnA uniqueidentifier, a int);
go
insert t1_2011 (a) values (1), (2)
go
insert myTable2 select newid(), a from t1_2011
go
-- should be equal to 2
select count(distinct a) from myTable2
go

drop table myTable2;
drop table myTable;
drop table t1_2011;
go