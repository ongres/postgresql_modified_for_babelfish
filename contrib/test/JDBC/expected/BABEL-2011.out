USE master;
GO

-- test volatility of functions in multi-row insert
-- newsequentialid()
CREATE TABLE myTable (ColumnA uniqueidentifier DEFAULT NEWSEQUENTIALID(), a int);
go
insert myTable (a) values (1), (2)
go
~~ROW COUNT: 2~~

-- should be equal to 2
select count(distinct a) from myTable
go
~~START~~
int
2
~~END~~


-- newid()
CREATE TABLE t1_2011 (a int);
CREATE TABLE myTable2 (ColumnA uniqueidentifier, a int);
go
insert t1_2011 (a) values (1), (2)
go
~~ROW COUNT: 2~~

insert myTable2 select newid(), a from t1_2011
go
~~ROW COUNT: 2~~

-- should be equal to 2
select count(distinct a) from myTable2
go
~~START~~
int
2
~~END~~


drop table myTable2;
drop table myTable;
drop table t1_2011;
go
